pkg/kubelet/kubelet_pods.go

```diff
@@ -425,6 +432,15 @@ func (kl *Kubelet) GetPodCgroupParent(pod *v1.Pod) string {
 	return cgroupParent
 }
 
+// GenerateRunContainerEnvs generates the environment variables for runtime container
+func (kl *Kubelet) GenerateRunContainerEnvs(pod *v1.Pod, container *v1.Container, podIP string) ([]kubecontainer.EnvVar, error) {
+	envs, err := kl.makeEnvironmentVariables(pod, container, podIP)
```

Comment:
Can we just keep the original `makeEnvironmentVariables` call instead of creating a new function to wrap it?

Reply:
@msau42 Yes makes sense. I'll squash and commit in the morning.  Is there any update on the e2e tests?

Reply:
Yes, node team is working on changing the test config so that it doesn't run alpha features.